controllers/temporalnamespace_controller.go

```diff
@@ -136,13 +140,126 @@ func (r *TemporalNamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		}
 	}
 
+	err = r.reconcileCustomSearchAttributes(ctx, &logger, namespace, cluster)
```

Comment:
@alexandrevilain is this the right place to start reconciling custom search attribtues?

Reply:
I think I should wait for the namespace to be marked ready, before reconciling custom search attributes, right? But where should I put that waiting logic?

Reply:
I think my current implementation is causing the `TemporalNamespace_ready` e2e test to fail